c/scheduling.c

```diff
@@ -35,6 +35,8 @@
 #include "recovery.h"
 #include "scheduling.h"
 #include "openprims.h"
+#include "logging.h"
```

Comment:
i'm like 80% sure that scheduling.c is not something we can use zowelog in. what do others think?